frontend/src/components/editor/actions/useCellActionButton.tsx

```diff
@@ -178,6 +183,18 @@ export function useCellActionButtons({ cell }: Props) {
           formatEditorViews({ [cellId]: editorView }, updateCellCode);
         },
       },
```

Comment:
When markdown is toggled, can we add a "View as Python" menu item, with the same hotkey?

Reply:
Yes, done